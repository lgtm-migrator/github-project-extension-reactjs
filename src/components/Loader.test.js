import React from 'react';
import { mount } from 'enzyme';
import Loader from './Loader';

describe('Loader', () => {
    it('matches snapshot', () => {
        const rendered = mount(<Loader />);
        expect(rendered).toMatchSnapshot()
    });
});