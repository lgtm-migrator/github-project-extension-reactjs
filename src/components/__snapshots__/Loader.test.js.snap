// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader matches snapshot 1`] = `
.c0 {
  width: 40px;
  height: 40px;
  background-color: #333;
  margin: 100px auto;
  -webkit-animation: jTDNZI 1.2s infinite ease-in-out;
  -webkit-animation: cDEMHD 1.2s infinite ease-in-out;
  animation: cDEMHD 1.2s infinite ease-in-out;
}

<Loader>
  <styled.div>
    <div
      className="c0"
    />
  </styled.div>
</Loader>
`;
